### DISCUSSION TITLE
Lab 4.01 task 1

### DISCUSSION THREAD

**Discussion by carlosdac55** [rramosp/ai4eng.v1#122] [04 - GESTIÓN DE DATOS]
Cordial saludo.

Realizando esta tarea, con el código que he escrito llego al resultado esperado, sin embargo, la calificación es 0 y no me da ninguna explicación porque esta nota. Quisiera saber si debo usar otros métodos o que me hace falta y no he visto para poder tener una buena nota.
Muchas gracias. 

![codigo](https://user-images.githubusercontent.com/85840609/167238462-c0985729-6622-4e2d-aa43-964e2e4ca541.JPG)

**Comment by carlosdac55** [comment]
Después de una segunda revisión, ya lo resolví.
Gracias

**Comment by rramosp** [comment]
perfecto! gracias por compartir

On Sat, May 7, 2022 at 12:49 AM Carlos Avalos ***@***.***>
wrote:

> Después de una segunda revisión, ya lo resolví.
> Gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/122#discussioncomment-2704875>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJA4G52HW6LJDLH5ATVIX743ANCNFSM5VJ22CGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
